src/neofs_testlib/cli/neofs_cli/shards.py

```diff
@@ -110,7 +110,6 @@ def list(
         self,
         endpoint: str,
         wallet: str,
```

Comment:
I guess if we don't need a password, then we don't need a wallet either.

Reply:
see https://github.com/nspcc-dev/neofs-testlib/pull/74#discussion_r1305908526